Automattic/woocommerce-payments
# Fix - Hide save payment method checkbox for non-reusable payment methods on WooCommerce Blocks

Fixes #3667 

#### Changes proposed in this Pull Request

This PR hides the option to save payment methods that cannot be reused on WooCommerce Checkout Blocks. e.g Giropay.

#### Testing instructions

* Follow steps in #3667 to replicate the problem.
* Checkout the branch `fix/3667-saving-payment-methods-upe` & build dependencies.
* Follow the steps again and confirm that save payment method checkbox is not displayed for non-reusable payment methods such as Giropay.

For successful checkout with blocks, #4104 needs to merged.

-------------------


**Post merge**
